chrome/content/coniks/context.js

```diff
@@ -6,56 +6,46 @@ Cu.import("chrome://otr/content/helpers.js");
 
 var OS;
 
-function ConiksContext(opts) {
-  if (!(this instanceof ConiksContext)) {
-    return new ConiksContext(opts);
-  }
-  opts = opts || {};
+class ConiksContext {
+  constructor(opts) {
+    opts = opts || {};
 
-  if (opts.account === null) {
-    return null;
-  }
-  this.account = opts.account;
-  if (opts.fingerprint !== undefined || opts.fingerprint !== null) {
-    this.fingerprint = opts.fingerprint;
+    if (opts.account === null) {
+      return null;
+    }
+    this.account = opts.account;
+    if (opts.fingerprint !== undefined || opts.fingerprint !== null) {
+      this.fingerprint = opts.fingerprint;
```

Comment:
This will be the public key fingerprint for contacts? But it seems like https://github.com/c633/ctypes-otr/blob/coniks-preferences/chrome/content/coniks/coniks.js#L52 sets the private key fingerprint for the account. It might be helpful to distinguish between the two more clearly.